app/mailers/alert_mailer.rb

```diff
@@ -20,7 +20,7 @@ def alert_email
     @title = @school.name
 
     email = if @batch_email
```

Comment:
I don't think you need this if statement anymore as the outcome is the same for both?

Reply:
yes, d'oh